From fc2eb1a484341fdcc42e0dacf4ec770fbc01092a Mon Sep 17 00:00:00 2001
From: Silke Hofstra <silke@slxh.eu>
Date: Fri, 6 May 2022 16:16:03 +0200
Subject: [PATCH 2/2] regress: export required symbols for compression test

Export the `fido_compress`, `fido_uncompress` and `fido_sha256` symbols
that are needed for the regression tests.
---
 src/export.gnu  | 3 +++
 src/export.llvm | 3 +++
 src/export.msvc | 3 +++
 3 files changed, 9 insertions(+)

diff --git a/src/export.gnu b/src/export.gnu
index 8fc94af..ca4b925 100644
--- a/src/export.gnu
+++ b/src/export.gnu
@@ -237,6 +237,9 @@
 		fido_init;
 		fido_set_log_handler;
 		fido_strerr;
+		fido_compress;
+		fido_uncompress;
+		fido_sha256;
 		rs256_pk_free;
 		rs256_pk_from_ptr;
 		rs256_pk_from_EVP_PKEY;
diff --git a/src/export.llvm b/src/export.llvm
index df9a667..7df941e 100644
--- a/src/export.llvm
+++ b/src/export.llvm
@@ -235,6 +235,9 @@ _fido_dev_largeblob_set_array
 _fido_init
 _fido_set_log_handler
 _fido_strerr
+_fido_compress
+_fido_uncompress
+_fido_sha256
 _rs256_pk_free
 _rs256_pk_from_ptr
 _rs256_pk_from_EVP_PKEY
diff --git a/src/export.msvc b/src/export.msvc
index a908215..9a86034 100644
--- a/src/export.msvc
+++ b/src/export.msvc
@@ -236,6 +236,9 @@ fido_dev_largeblob_set_array
 fido_init
 fido_set_log_handler
 fido_strerr
+fido_compress
+fido_uncompress
+fido_sha256
 rs256_pk_free
 rs256_pk_from_ptr
 rs256_pk_from_EVP_PKEY
-- 
2.35.3

